src/frontend/src/utils/design-review.utils.tsx

```diff
@@ -0,0 +1,110 @@
+import { DesignReview, DesignReviewStatus } from 'shared';
+import {
+  alfred,
+  aquaman,
+  batman,
+  flash,
+  greenlantern,
+  superman,
+  wonderwoman
+} from '../../../backend/tests/test-data/users.test-data';
+import ConstructionIcon from '@mui/icons-material/Construction';
+import WorkOutlineIcon from '@mui/icons-material/WorkOutline';
+import ElectricalServicesIcon from '@mui/icons-material/ElectricalServices';
+import TerminalIcon from '@mui/icons-material/Terminal';
+
+export const NOON_IN_MINUTES = 720;
+
+export enum DAY_NAMES {
+  Monday,
+  Tuesday,
+  Wednesday,
+  Thursday,
+  Friday,
+  Saturday,
+  Sunday
+}
```

Comment:
you've got two utils files rn, you should combine any methods/ stuff that aren't duplicates of each other into the .ts file, then if there are any methods that return React components just move them elsewhere or if they're used in multiple places, make a separate component file for them